test/controllers/signs_controller_test.rb

```diff
@@ -136,4 +136,38 @@ class SignsControllerTest < ActionController::TestCase
     # Assert
     assert_response :success
   end
+
+  test "assert that `controls muted` attributes are NOT present on slides with foreground_type: video on the `/play` page" do
```

Comment:
Try to give you tests more natural title/descriptions using "expects". For example:

    test "expects video in video slide to be muted with controls hidden on sign play page" do